icu4c/source/test/intltest/calregts.cpp

```diff
@@ -2747,7 +2747,7 @@ void CalendarRegressionTest::TestDeprecates(void)
 {
     UErrorCode status = U_ZERO_ERROR;
     Calendar *c1 = Calendar::createInstance("ja_JP@calendar=japanese",status);
-    Calendar *c2 = Calendar::createInstance("ja_JP_TRADITIONAL",status);
+    Calendar *c2 = Calendar::createInstance("ja_JP@calendar=japanese",status);
```

Comment:
I think it does not make much sense to keep this test case. I'd prefer to simply drop this test case.

Reply:
I think we should keep it. The point of this test is to make sure that using API with the C++ enum EDateField is equivalent to using API with the C enum UCalendarField. It just happened to also test with formerly equivalent locale IDs.

Reply:
OK, got it. I'm fine with keeping it.